relayer/cmd/parachain_head_proof.go

```diff
@@ -72,7 +66,8 @@ func ParachainHeadProofFn(cmd *cobra.Command, _ []string) error {
 	copy(beefyBlockHash[:], beefyBlockHashHex[0:32])
 
 	relayChainBlock, _ := cmd.Flags().GetUint64("relaychain-block")
-	mmrProof, err := conn.GenerateProofForBlock(relayChainBlock, beefyBlockHash)
+	// magic plus 1 to make the block a leaf index
```

Comment:
Is this command used in prod or is it a convenience method for testing?

Reply:
Convenience for testing.